jbpm-flow-builder/src/main/java/org/jbpm/osgi/flow/compiler/Activator.java

```diff
@@ -35,7 +36,8 @@ public void start(BundleContext bc) throws Exception {
         this.processBuilderReg = bc.registerService( new String[]{ ProcessBuilderFactoryService.class.getName(), Service.class.getName()},
                                                                    new ProcessBuilderFactoryServiceImpl(),
                                                                    new Hashtable() );
-        ProcessBuilderFactory.reInitializeProvider();
+        RuntimeManagerFactory.Factory.reset();
+        ProcessBuilderFactory.reInitializeProvider();        
```

Comment:
Could we use the same method name, co be consistent? The `reInitializeProvider()` may not be ideal, but we could change that as well potentially.